build.gradle

```diff
@@ -26,9 +26,9 @@ buildscript {
     }
 
     dependencies {
-        classpath "com.android.tools.build:gradle:${providers.systemProperty("agpVersion").forUseAtConfigurationTime().orNull ?: '4.1.3'}"
-        classpath 'com.google.gms:google-services:4.2.0'
-        classpath 'com.google.android.gms:oss-licenses-plugin:0.9.4'
+        classpath "com.android.tools.build:gradle:${providers.systemProperty("com.android.gradle.version").forUseAtConfigurationTime().orNull ?: '4.1.3'}"
```

Comment:
I would like to keep the `agpVersion` as a property to select the AGP plugin version. Is that possible?

Reply:
Yes, I think this is the only usage of this system property, so feel free to rename/change it.

Reply:
Renamed back to `agpVersion`.